README.md

```diff
@@ -12,20 +12,24 @@ The above mentioned feature list is elaborated here:
     - Required Fields:
       - full name
       - email id
-      - password
+      - password (must be stored in encrypted format)
+      - Send JWT
```

Comment:
There is no need for this - as the intent is to show the fields that are required from Frontend